guides/common/modules/proc_changing-the-default-mirroring-policy.adoc

```diff
@@ -0,0 +1,36 @@
+[id="Changing_the_Default_Mirroring_Policy"]
```

Comment:
```suggestion
:_mod-docs-content-type: PROCEDURE

[id="changing-the-default-mirroring-policy"]
```

Two suggestions: Add the module type prefix similar to all other modules, and use an anchor that is similar to the file name which makes grepping much easier.

Reply:
I've made the changes to the new mirroring policy file and also the older default download policy file that I based it on.